polaris/viz/spherical.py

```diff
@@ -2,17 +2,116 @@
-                      colorbar_label=None):
+def plot_global_mpas_field(mesh_filename, da, out_filename, config,
+                           colormap_section, title=None,
+                           plot_land=True, colorbar_label='',
+                           central_longitude=0., figsize=(8, 4.5),
+                           dpi=200):
+    """
+    Plots a data set as a longitude-latitude map
+
+    Parameters
+    ----------
+    mesh_filename : str
+        A filename containing the MPAS mesh
+
+    da : xarray.DataArray
+        The horizontal field to plot
+
+    out_filename : str
+        The image file name to be written
+
+    config : polaris.config.PolarisConfigParser
+        The config options to use for colormap settings
+
+    colormap_section : str
+        The name of a section in the config options. Options must include:
+
+        colormap_name
+            The name of the colormap
+
+        norm_type
+            The norm: {'linear', 'log'}
+
+        colorbar_limits
+            The minimum and maximum value of the colorbar
+
+    title : str, optional
+        The subtitle of the plot
+
+    plot_land : bool
+        Whether to plot continents over the data
+
+    colorbar_label : str, optional
+        Label on the colorbar
+
+    central_longitude : float, optional
+        The longitude of the center of the plot
+
+    figsize : tuple, optional
+        The size of the figure in inches
+
+    dpi : int, optional
+        Dots per inch for the output plot
+    """
+    matplotlib.use('agg')
+    uxgrid = ux.open_grid(mesh_filename)
+    uxda = ux.UxDataArray(da, uxgrid=uxgrid)
+
+    gdf_data = uxda.to_geodataframe()
+
+    hv.extension('matplotlib')
+
+    projection = cartopy.crs.PlateCarree(central_longitude=central_longitude)
+
+    colormap = config.get(colormap_section, 'colormap_name')
+
+    norm_type = config.get(colormap_section, 'norm_type')
+    if norm_type == 'linear':
+        logz = False
+    elif norm_type == 'log':
+        logz = True
+    else:
+        raise ValueError(f'Unsupported norm_type: {norm_type}')
+
+    colorbar_limits = config.getlist(colormap_section, 'colorbar_limits',
+                                     dtype=float)
+
+    plot = gdf_data.hvplot.polygons(
+        c=da.name, cmap=colormap, logz=logz,
+        clim=tuple(colorbar_limits),
+        clabel=colorbar_label,
+        width=1600, height=800, title=title,
+        xlabel="Longitude", ylabel="Latitude",
+        projection=projection,
+        rasterize=True)
+
+    if plot_land:
+        plot = plot * gf.land * gf.coastline
+
+    plot.opts(cbar_extend='both', cbar_width=0.03)
+
+    fig = hv.render(plot)
+    fig.set_size_inches(figsize)
+    fig.savefig(out_filename, dpi=dpi, bbox_inches='tight', pad_inches=0.1)
+
+
+def plot_global_lat_lon_field(lon, lat, data_array, out_filename, config,
+                              colormap_section, title=None, plot_land=True,
+                              colorbar_label=None):
```

Comment:
@xylar - I think it's good to leave in the existing capability to plot on a remapped lat/lon grid. I'm just wondering if the holowview approach is efficient enough to replace it more generally, i.e. for MPAS-Analysis purposes?

Reply:
It might be.  @cbegeman and I were talking about this.  We currently do analysis on a comparison grid and that sometimes makes it easier to compare with observations or with other model runs on a different mesh.  But there's not really a reason we couldn't do as much as possible on the native mesh.

Reply:
Good point about using a comparison grid for in those cases. That makes sense. I was just curious.